src/goto-symex/postcondition.cpp

```diff
@@ -174,7 +174,7 @@ bool postconditiont::is_used(
     // aliasing may happen here
 
     value_setst::valuest expr_set;
-    value_set.get_value_set(expr.op0(), expr_set, ns);
+    value_set.read_value_set(expr.op0(), expr_set, ns);
```

Comment:
I'm only starting to review this, so please forgive me if I'm asking a dumb question: is this renaming essential?

Reply:
These were renamed because in an earlier version of this PR, the 4-arg version of `get_value_set` was virtual but the 3-arg version wasn't, which was annoying as you had to manually forward it in each derived class, hence renaming the overload out of the way. However since that has changed, I'll remove these.